conans/model/profile.py

```diff
@@ -86,9 +77,9 @@ def get_package_name_value(item):
                         raise ConanException("Invalid env line '%s'" % env)
                     package_name, name, value = get_package_name_value(env)
                     if package_name:
-                        obj._package_env[package_name][name] = value
+                        obj.env_values.add(name, value, package_name)
```

Comment:
You upvoted the comment, but lately didn't move this to EnvValues. Forgot about it or something else I am missing?